src/satosa/micro_services/consent.py

```diff
@@ -176,8 +176,9 @@ def _consent_registration(self, consent_args):
         :return: Ticket received from the consent service
         """
         jws = JWS(json.dumps(consent_args), alg=self.signing_key.alg).sign_compact([self.signing_key])
-        request = "{}/creq/{}".format(self.api_url, jws)
-        res = requests.get(request)
+        payload = dict(jwt=jws)
+        request = "{}/creq/_".format(self.api_url)
```

Comment:
It is a bit ugly to use `.../creq/_` – the `_` part. Can we drop it and make be just `.../creq/`?

Reply:
No, the router in CMService will not match...